fastavro/_logical_writers.pyx

```diff
@@ -33,6 +35,7 @@ cpdef prepare_timestamp_millis(object data, schema):
     cdef object tt
     cdef tm time_tuple
     if isinstance(data, datetime.datetime):
+        print(f'date: {data}')
```

Comment:
Please remove this

Reply:
Oh wow, sorry about this. I noticed + removed it when doing the Linux tests, but forgot to push.
I'll attend to the other remarks as well.